src/lib/y2network/network_manager/connection_config_writers/base.rb

```diff
@@ -105,12 +105,12 @@ def configure_dhcp(conn)
         # Convenience method to configure the reference to the parent or master device
         #
         # @param parent [Y2Network::ConnectionConfig::Base] Connection to take settings from
-        def configure_as_child(parent)
-          slave_type = "bridge" if parent.type.br?
-          slave_type = "bond" if parent.type.bonding?
-          return unless slave_type
+        def configure_as_port(parent)
+          port_type = "bridge" if parent.type.br?
+          port_type = "bond" if parent.type.bonding?
+          return unless port_type
 
-          file.connection["slave-type"] = slave_type
+          file.connection["slave-type"] = port_type
```

Comment:
```suggestion
          # slave-type is NetworkManager API
          file.connection["slave-type"] = port_type
```

I have checked the NM git repo, so far the code has slaves, there is no PR, but there is an Issue
https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/779 "changing problematic language in nmcli" which says that around 2022-07 they should have inclusive naming

Reply:
Correction: something got merged for NM 1.34.0, apparently only in the D-Bus API, not in config files.
https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/merge_requests/985/diffs

Reply:
ok, thanks